vss-processor/src/main/kotlin/org/eclipse/kuksa/vssprocessor/VssDefinitionProcessor.kt

```diff
@@ -75,7 +76,7 @@ class VssDefinitionProcessor(
             codeGenerator.createNewFile(
                 Dependencies(false, containingFile),
                 PACKAGE_NAME,
-                DEFAULT_FILE_NAME,
+                classDeclaration.toClassName().simpleName + FILE_NAME_PROCESSOR_POSTFIX,
```

Comment:
Could you shortly elaborate the purpose of this change? 

classDeclaration should be the class which is annotated by the VssDefinition class, so in our case it would be KuksaDataBrokerActivityProcessor...?

Reply:
Correct. It eliminates an issue where you could annotate two different classes and kept will then throw an error.

Reply:
Understood, but will it then also process everything twice, because it found the same annotation (with same or different spec file) twice? 
Expectation: That should not work and should in fact throw a meaningful exception.

Reply:
If the specification has the same information for a model then it will throw an exception. If it does not (an extension file for example) it should work (did not test that one yet). We will probably have to support that in the near future anyway and can test it then and create a ticket if something is missing.